docs/proposals/20190610-machine-states-preboot-bootstrapping.md

```diff
@@ -139,7 +139,7 @@ type MachineSpec struct
         - Type: `Bootstrap`
         - Description: Bootstrap is a reference to a local struct which encapsulates fields to configure the Machine’s bootstrapping mechanism.
     - **InfrastructureRef [required]**
-        - Type: `corev1.TypedLocalObjectReference`
+        - Type: `corev1.ObjectReference`
```

Comment:
Why the switch to ObjectReference vs TypedLocalObjectReference here?

Reply:
Typed doesn't have the api version